device_plugins/deploy_qat.md

```diff
@@ -37,6 +37,7 @@ $ oc describe node <node name> | grep qat.intel.com
  qat.intel.com/dc: 32 
  qat.intel.com/dc: 32 
  ```
+**Note**: By default the device plugin registers half resources each for `qat.intel.com/cy` and `qat.intel.com/dc` respectively. For more details about the kernel driver reset configuration, refer to the section below.
```

Comment:
for more detail about the QAT resources configuration, please see section QAT Device Plugin Configuration section below